docs/docker-integration/index.rst

```diff
@@ -0,0 +1,92 @@
+.. _docker-integration:
+
+======================
+Docker, Swarm, Compose
+======================
+
+Flocker works with Docker, Swarm and/or Compose via the Flocker Plugin for Docker.
+Follow an installation guide to get started:
+
+Flocker Installation Options
+============================
+.. raw:: html
+
+	<div class="pods-eq">
+	    <div class="pod-boxout pod-boxout--2up pod-boxout--recommended">
+		   <img src="../_images/amazon-docker2x.png" alt="Amazon AWS logo"/>
+		   <span>Use our CloudFormation template to install Flocker on AWS<em>Fastest</em></span>
```

Comment:
:question: Perhaps "Use our CloudFormation template to install Flocker, Docker Swarm and Docker Compose on AWS.